contrib/runners/cloudslang_runner/cloudslang_runner/runner.yaml

```diff
@@ -14,3 +14,8 @@
       description: Action timeout in seconds. Action will get killed if it doesn't
         finish in timeout seconds.
       type: integer
+  output_schema:
+    stdout:
+      type: "string"
+    stdout:
```

Comment:
Copy/paste error? I imagine that the second key should be `stderr`.

Reply:
Good catch.